src/tribol/mesh/MfemData.cpp

```diff
@@ -609,7 +609,7 @@ void MfemMeshData::UpdateData::SetElementData()
     num_verts_per_elem_ = mfem::Geometry::NumVerts[element_type];
   } else {
     // just put something here so Tribol will not give a warning for zero element meshes
-    elem_type_ = LINEAR_EDGE;
+    elem_type_ = LINEAR_QUAD;
```

Comment:
Perhaps enhance the comment to currently put the surface topology (we don't support volume interfaces yet) with the max number of nodes?